### PR TITLE
[RFR] add a new exported to have a more usable JSON (compacted version)

### PR DESCRIPTION
**Pull request by kerphi**

### CODE REVIEW

**Comment by ThieryMichel** [comment]
need rebase

File: src/api/exporters/ezsLocals.js [hunk 1]
```diff
@@ -103,3 +103,17 @@ export function JSONLDString(data, feed) {
...
+    } else {
+        jsonld.compact(data, {})
+            .then((out) => {
+                feed.send(out);
+            },
+            (err) => {
+                throw err;
+            });
+    }
+}
```
Comment (ThieryMichel) [inline src/api/exporters/ezsLocals.js]: add a linefeed

**Review by ThieryMichel — DISMISSED** [review]

**Comment by ThieryMichel** [comment]
do `git rebase -i master`, and remove (comment them with `#`) my already merged commit : 
- make dataset column bold and black
- display composed of indication on field header in preview
- fix nquads and turtle exporters

**Comment by kerphi** [comment]
j'ai fait git rebase -i master et j'ai ceci, je pige pas l'étape suivante ?
image

**Comment by ThieryMichel** [comment]
Ok, have you done a git pull on master before?

**Comment by kerphi** [comment]
yes :-/

**Pull request merged**